package.json

```diff
@@ -69,19 +69,27 @@
     "medium-zoom": "^1.0.6",
     "opencollective-postinstall": "^2.0.2",
     "prismjs": "^1.23.0",
+    "solid-jest": "^0.2.0",
+    "solid-js": "^1.3.1",
     "strip-indent": "^3.0.0",
     "tinydate": "^1.3.0",
     "tweezer.js": "^1.4.0"
   },
   "devDependencies": {
     "@babel/core": "^7.11.6",
     "@babel/preset-env": "^7.11.5",
+    "@rollup/plugin-babel": "^5.3.0",
+    "@rollup/plugin-commonjs": "^21.0.1",
+    "@rollup/plugin-node-resolve": "^13.1.3",
+    "@rollup/plugin-replace": "^3.0.1",
     "autoprefixer-stylus": "^1.0.0",
     "axios": "^0.21.1",
     "babel-eslint": "^10.0.3",
-    "babel-jest": "^26.3.0",
+    "babel-jest": "^27.0.0",
+    "babel-preset-solid": "^1.3.1",
     "browser-sync": "^2.26.12",
     "chokidar": "^3.4.2",
+    "colors": "^1.4.0",
```

Comment:
Pinned because `live-server` depends on libraries using `@latest` which is a good way to get breaking changes into projects, including this one: https://github.com/Marak/colors.js/issues/285